### ISSUE TITLE
Create tables and API for session data

### ISSUE THREAD

**Issue by gunar** [theopensystemslab/planx-new#31]
![image](https://user-images.githubusercontent.com/7684574/92770388-cfc7b280-f391-11ea-9180-cd1d41f1cdee.png)

### Public interface (PI) flow

1. User visits page
1. User enters area code
1. PI downloads passport data
1. PI sends GraphQL request to create session, gets back a session id
    ```gql
    mutation CreateSession {
      insert_session_one(object: {
        id: uuid,
        flow_id: uuid,
        flow_version: integer,
        flow_data: jsonb,
        passport: jsonb,
      }) {
        id
      }
    }
    ```
1. Once each question is answered, PI sends GraphQL requests to create session events
    ```gql
    mutation CreateSessionEvent {
      insert_session_event(objects: {
        session_id: uuid, # same as before
        chosen_node_ids: [uuid],
        type: ENUMB(human_decision|automatic_decision),
      }) {
        affected_rows
      }
    }
    ```
1. Once the last question is answered, PI sends a GraphQL request to end the session
    ```gql
    mutation EndSession {
      update_session_by_pk(pk_comlumns: {id: {_eq: uuid}}, _set: {completed_at: NOW()}) {
        id
      }
    }
    ```

### Database schema
- Basically [this image](https://files.slack.com/files-pri/T03T7JTFV-F01A8FFPSDC/screenshot_2020-09-10_at_4.22.45_pm.png)

### Hasura (i.e. backend)

- Create sessions and session_events tables
- Create access control rules (ACLs) so that users can only:
    - INSERT into session
    - UPDATE session.completed_at
    - INSERT into session_event
